tests/test_with_fixtures.py

```diff
@@ -0,0 +1,77 @@
+from pytest_fun.funs import write_int_to_file, sum_dict
+import pytest
+from pathlib import Path
+
+
+# ~~~~ test write_int_to_file ~~~~
+@pytest.mark.use_fixtures
+def test_write_int_to_file():
+    # define a temporary directory
+    tmpdir = Path('/tmp')
```

Comment:
I'm confused ... this is a normal test not using a fixture? So just an example of defining `tmpdir` yourself ... though you can use the `tmpdir` fixture from pytest.

Reply:
I didn't think about that. Thanks. I will merge this PR first and, then, create a new version.